instance.go

```diff
@@ -461,22 +480,22 @@ func (i *Instance) ExecuteWithOutput(namespace string, codeReader io.Reader, out
 	routineName := filepath.Base(codePath)
 	defer i.removeTempRoutine(namespace, routineName)
 
-	cmd := i.GetCSessionCommand(namespace, "EnsLibMain^"+routineName)
+	cmd := i.GetSessionCommand(namespace, "EnsLibMain^"+routineName)
 
 	cmd.Stdout = out
 	if err := cmd.Start(); err != nil {
-		log.WithError(err).Debug("Failed to start csession")
+		log.WithError(err).Debug("Failed to start session")
 		return err
 	}
 
-	elog.Debug("Waiting on csession to exit")
+	elog.Debug("Waiting on session to exit")
 	return cmd.Wait()
 }
 
-// GetCSessionCommand will return a properly configured instance of exec.Cmd to
-// run the provided command (properly formatted for csession) in the provided
+// GetSessionCommand will return a properly configured instance of exec.Cmd to
+// run the provided command (properly formatted for session) in the provided
 // namespace.
-func (i *Instance) GetCSessionCommand(namespace, command string) *exec.Cmd {
+func (i *Instance) GetSessionCommand(namespace, command string) *exec.Cmd {
```

Comment:
I'd probably drop the `Get` from this and similar function names since it's already a break. However, I don't feel strongly about this.